zboxcore/zboxutil/http.go

```diff
@@ -542,9 +543,18 @@ func MakeSCRestAPICall(scAddress string, relativePath string, params map[string]
 				responses[response.StatusCode]++
 				if responses[response.StatusCode] > maxCount {
 					maxCount = responses[response.StatusCode]
+				}
+
+				if responses[response.StatusCode] > maxCount {
+					maxCount = responses[response.StatusCode]
+				}
+
+				if isCurrentDominantStatus(response.StatusCode, responses, maxCount) {
```

Comment:
Presumably we will also want to do this on every other http call that requires consensus, including calls to miners?

Reply:
@stewartie4 I did not look into other parts as I am not sure what's are the "actual rules". 

At one point, I actually thought that it should not always dominant response that has to be used. For example, if threshold consensus is 25%, if 1/4 is 200, then maybe that is what we are to use even with dominant is 404. FYI @dabasov @peterlimg 

The PR is just to fix what "seems" intended by function by the current implementation.

Reply:
No we should not do that.
If 3 out of 4 responses are 404 then we should return 404 error.
cc @peterlimg